examples/knowledgebasebot/data/nlu.md

```diff
@@ -0,0 +1,58 @@
+## intent:greet
+- hey
+- hello
+- hi
+- good morning
+- good evening
+- hey there
+
+## intent:goodbye
+- bye
+- goodbye
+- see you around
+- see you later
+
+## intent:query_knowledge_base
+- what [restaurants](object_type:restaurant) can you recommend?
+- list some [restaurants](object_type:restaurant)
+- can you name some [restaurants](object_type:restaurant) please?
```

Comment:
is `object_type` necessary or would `type` suffice?

Reply:
I would like to keep `object_type`. I think `type` is too generic. Would not be clear from the name that it is related to the knowledge base object.